loggerFactory.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const pino = require('pino');
+const Logger = pino({}, pino.destination("./tmp/proxy.log"));
+
+Logger.level = 'debug';
```

Comment:
Make it info when going on prod or put behind an env flag

Reply:
added